### DISCUSSION TITLE
Prevent launching onedrive at startup

### DISCUSSION THREAD

**Discussion by aletgn** [abraunegg/onedrive#1418] [Q&A] [answered]
Hi everyone,
it has been a week that I'm using OneDrive client. It works great. I have been running the latest version of the client (downloaded from ppa ``yann1ck'') on an Ubuntu 20.04 machine. I haven't touched the configuration, thus the client has been working with the default one so far.

Is there a way to **prevent** OneDrive client to be launched at the startup of the operating system?

I've already read the documentation, but I can't figure out how to do it.

Thank you
Best regards

**Comment by abraunegg** [comment, marked as answer]
@aletgn 
> Is there a way to prevent OneDrive client to be launched at the startup of the operating system?

Please refer to this bug that is documented including how to fix this issue: https://github.com/abraunegg/onedrive/issues/1274

**Reply by aletgn** [reply]
I am very sorry to have posted in the wrong section, I am a beginner.

Thanks for the reference you have indicated. I will try to do what is described there.
